prebid-mobile/pbm-api/iOS/code-integration-ios.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page_v2
+title: Code Integration - iOS
+description: Code Integration - iOS
+pid: 1
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-ios
+sidebarType: 2
+---
+
+
+
+
+# Code Integration for iOS
+
+Get started with Prebid Mobile by creating a [Prebid Server account]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+### Use Cocoapods?
+
+Easily include the Prebid Mobile SDK for your primary ad server in your Podfile.
+
+```
+platform :ios, '11.0'
+
+target 'MyAmazingApp' do
+    pod 'PrebidMobile'
+end
+```
+
+### Build framework from source
+
+Build Prebid Mobile from source code. After cloning the repo, from the root directory run
+
+```
+./scripts/buildPrebidMobile.sh
+```
+to output the PrebidMobile.framework.
+
+### Integrate Ad Servers With Your App
+
+Integrating **MoPub** with your application
+1.  Go to [MoPub.com](https://app.mopub.com/account/register/) and  register for a MoPub account . If you already have an account with them, you can [log in](https://app.mopub.com/account/login/) . 
+2.  After the registration you will be automatically prompted to set up a new MoPub application required for integrating mobile ads to your application.
+
+Integrating **Google** with your application   
+
+Go to Google's developer site and follow the instructions for integrating their [Mobile Ads SDK](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/quick-start) into your app.
+
```

Comment:
First we need to provide prebid server details like serverId, prebid host before creating adUnits. This is a required section